modules/cache-material/test/clone_cryptographic_material.test.ts

```diff
@@ -87,13 +87,12 @@ describe('cloneMaterial', () => {
   })
 
   it('clone WebCryptoDecryptionMaterial', () => {
+    /* WebCryptoDecryptionMaterial do not have an unencrypted data key. */
     const material = new WebCryptoDecryptionMaterial(webCryptoSuite, { some: 'context' })
-      .setUnencryptedDataKey(udk128, trace)
```

Comment:
I know this change fixes the cloning so that it handles the WebCryptoDecryptionMaterials case, but is setting an unencryptedDataKey on WebCryptoDecryptionMaterials something that should be allowed in the first place?

Reply:
This may be true.  I'd have to think about it.  But it would also be a VERY large change.  Having it _should_ not hurt anything...

Reply:
If there is a potential for this being a sharp edge, can we either document this somewhere (maybe CryptoKey or WebCryptoDecryptionMaterial) or create an issue for further investigation?

Reply:
Discussed offline, we do not see this as a sharp edge. Investigation for how to make this easy to maintain is part of https://github.com/awslabs/aws-encryption-sdk-javascript/issues/74